src/components/calendar/CalendarPanel.js

```diff
@@ -18,7 +18,7 @@ export class CalendarPanel extends Component {
 
     renderElement() {
         return (
-            <div ref={(el) => this.element = el} className={this.props.className} style={this.props.style}>
+            <div ref={(el) => {this.element = el; if(this.props.forwardRef) this.props.forwardRef(el)}} className={this.props.className} style={this.props.style}>
```

Comment:
this.element is not used anywhere as far as I can tell, maybe it can be removed.